### DISCUSSION TITLE
无法通过get_msg获取 [CQ:reply]的消息的id

### DISCUSSION THREAD

**Discussion by ilyfairy** [Mrs4s/go-cqhttp#2205] [Q&A]
`[CQ:reply,id=123456]`, 这个id无法通过get_msg获取消息内容  
我无法通过这个id获取到用户回复的消息是什么  

我有一个这样的需求:  
bot发送了三条消息, 储存消息id, 让用户回复  
然后bot接收到了`[CQ:reply,id=xxx]` , 通过这个id让bot知道用户回复了哪条消息  
但是发现send_group_msg返回的消息id和reply的id没有任何关联

**Comment by xiangxiangxiong9** [comment]
send_group_msg 返回的消息id是通过此api发送后的消息id，与原消息无关
replay的那个的话，把 config.yml 里的 message.extra-reply-data 设置为 true 看看?
